grpc-spring-boot-starter/src/main/java/org/lognet/springboot/grpc/security/GrpcServiceAuthorizationConfigurer.java

```diff
@@ -74,8 +74,13 @@ private AuthorizedMethod(ServiceDescriptor... serviceDescriptor) {
         }
 
         public GrpcServiceAuthorizationConfigurer.Registry hasAnyAuthority(String... authorities) {
-            for (String auth : authorities) {
-                GrpcServiceAuthorizationConfigurer.this.registry.map(auth, methods);
+            if (authorities.length == 0) {
```

Comment:
Please move this logic to `processSecuredAnnotation()` method (call `authenticated()` if roles array is empty).
The reason for this is semantic

Reply:
OK, got it.
 I've add comment, please also wright few words about this behavior in README and add test for this case.
Thanks.

Reply:
Could you please add the "normal" value behavior to the readme. I don't fully understand it yet.